pilot/pkg/proxy/envoy/xds/xds.go

```diff
@@ -34,10 +33,10 @@ import (
 	"istio.io/istio/pkg/config/schema/collections"
 )
 
-type Server struct {
+type SimpleServer struct {
```

Comment:
xds.go is confusing here now, we can rename it afterward

Reply:
Yes. I agree. Will do in follow-up